graphs/docs/usage/getting_started.rst

```diff
@@ -1,15 +1,16 @@
 .. _usage-getting-started:
 
```

Comment:
Nice section! But it could it be worth that "Getting started" comes after "overview" rather than "Building Graphs". It seems like a more introductory/entry section so if it's someone looking to start using graphs it might be easier to find and then after that jump to a more advance section like "building graphs"